static/app/views/settings/project/projectKeys/details/keySettings.tsx

```diff
@@ -267,14 +301,34 @@ export function KeySettings({onRemove, organization, params, data}: Props) {
                       label={value.label}
                       key={key}
                       name={key}
-                      value={dynamicSDKLoaderOptions[sdkLoaderOption]}
+                      value={
+                        value.requiresV7 &&
+                        !sdkVersionSupportsPerformanceAndReplay(browserSdkVersion)
+                          ? false
+                          : dynamicSDKLoaderOptions[sdkLoaderOption]
+                      }
                       onChange={() =>
                         handleToggleDynamicSDKLoaderOption(
                           sdkLoaderOption as DynamicSDKLoaderOption,
                           !dynamicSDKLoaderOptions[sdkLoaderOption]
                         )
                       }
-                      disabled={!hasAccess}
+                      disabled={
+                        !hasAccess ||
+                        (value.requiresV7 &&
+                          !sdkVersionSupportsPerformanceAndReplay(browserSdkVersion))
+                      }
+                      help={
+                        value.requiresV7 &&
+                        !sdkVersionSupportsPerformanceAndReplay(browserSdkVersion)
+                          ? t('Only available in SDK version 7.x and above')
+                          : key === DynamicSDKLoaderOption.HAS_REPLAY &&
+                            dynamicSDKLoaderOptions[sdkLoaderOption]
+                          ? t(
+                              'When using Replay, the loader will load the ES6 instead of the ES5 bundle.'
```

Comment:
```suggestion
                              'When using including Replay or Performance, the loader will load the ES6 bundle instead of the ES5 bundle.'
```

Reply:
We need a case for performance as well

Reply:
that only restricts to V7, not ES6!

Reply:
Ah true, good point!